subprojects/model-core/src/testFixtures/groovy/org/gradle/internal/reflect/validation/ValidationMessageChecker.groovy

```diff
@@ -166,12 +166,16 @@ trait ValidationMessageChecker {
     )
     String incorrectUseOfInputAnnotation(@DelegatesTo(value = IncorrectUseOfInputAnnotation, strategy = Closure.DELEGATE_FIRST) Closure<?> spec = {}) {
         def config = display(IncorrectUseOfInputAnnotation, 'incorrect_use_of_input_annotation', spec)
-        config.description("has @Input annotation used on property of type '${config.propertyType}'")
+        def incorrectUseOfInputAnnotation = config.description("has @Input annotation used on property of type '${config.propertyType}'")
```

Comment:
Why is this change here?

Reply:
It's because for Directory and DirectoryProperty we have different suggestions in case of incorrect usage on `@Input`. Do you think we should create a different method due to that?

Reply:
I think it makes sense to have different suggestions. What I don't understand why this change is in the current PR.

Reply:
I see. I used this method for catching Sentry Android plugin issues with incorrect inputs. I upgraded plugins in [this commit](https://github.com/gradle/gradle/pull/21481/commits/39b9477c9ee16f64cb1d56230c0180ee69027a0f), since some were not compatible with AGP 7.3+ and after merging the latest changes from master this issue popped up.

Handling of Sentry plugin errors that contain also this change were made in [this commit](https://github.com/gradle/gradle/pull/21481/commits/8908fb5a42cf89f351cf1a5f0fb212b305b62c4d).